src/Symfony/Bundle/FrameworkBundle/Tests/Templating/TemplateNameParserTest.php

```diff
@@ -86,4 +81,39 @@ public function testParseValidNameWithNotFoundBundle()
     {
         $this->parser->parse('BarBundle:Post:index.html.php');
     }
+
+    /**
+     * @group legacy
+     * @dataProvider provideAbsolutePaths
+     */
+    public function testAbsolutePathsAreDeprecated($name, $logicalName, $path, $ref)
+    {
+        $deprecations = array();
+        set_error_handler(function ($type, $msg) use (&$deprecations) {
+            if (E_USER_DEPRECATED === $type) {
+                $deprecations[] = $msg;
+            }
```

Comment:
just for safety, maybe we should do something in the "else" case? like forwarding to the previous error handler?